src/Microsoft.DncEng.Configuration.Extensions/MappedJsonConfigurationProvider.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using Microsoft.Extensions.Configuration.Json;
+
+namespace Microsoft.DncEng.Configuration.Extensions
+{
+    public class MappedJsonConfigurationProvider : JsonConfigurationProvider
+#if NETCOREAPP2_1
+        , IDisposable
```

Comment:
? Why only disposable in app 2_1?

Reply:
Because "standard" is going to make this... weird... your runtime might be netcoreapp2.1, but your build might be whatever random version of standard that is.  What in the world is that going to do?

Reply:
What do you mean? This isn't built `standard`. This IDisposable is require on 2.1 because JsonConfigurationProvider was made IDisposable in 3.1.

Reply:
Can't you just do it in both places?  Though the base call will still break.  I don't have a good solution.  This should have an #error clause if "netstandard" is defined, because it cannot work there.

Reply:
If I reimplement IDisposable then I remove the dispose done in the base class.

This project isn't compiled for netstandard, should I add an #error for it anyways so we don't mess it up in the future?

Reply:
Yeah, I'd add the error. It's to tempting for someone to just add "netstandard" into the `<Frameworks>` element in the future.  After all, it's standard, why wouldn't it work? :-)